src/emerge.cpp

```diff
@@ -468,7 +468,7 @@ bool EmergeManager::pushBlockEmergeData(
 bool EmergeManager::popBlockEmergeData(v3s16 pos, BlockEmergeData *bedata)
 {
 	std::map<v3s16, BlockEmergeData>::iterator it;
-	std::unordered_map<u16, u16>::iterator it2;
+	std::unordered_map<u16, u32>::iterator it2;
```

Comment:
If you're editing this anyway please move `it` and `it2` inline with `auto` as type